react/react2/week1/user-searcher/src/ContextForRepos.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState, useEffect } from "react";
+import { createContext } from "react";
+
+export const ContextForRepos = createContext();
+
+export default function ReposProvider({ children }) {
```

Comment:
nice Haile, but you could have put all the states in the context file 
its will be more clear and less complicated for u